### PR TITLE
feat: update return method component in swap and return modal

### PR DESCRIPTION
**Pull request by sebastiannicolajsen**
### What 
- Updated the return method component of the create swap and request return modals. 

### Why
- To unify these with the create claim modal.

### How
- Reuse the claim component across the two other modals and ensure that it updates the refundable amount (request return) and total balance (create swap)

### CODE REVIEW

File: src/domain/orders/details/returns/index.js [hunk 1]
```diff
@@ -30,12 +32,27 @@ const ReturnMenu = ({ order, onReturn, onDismiss, toaster }) => {
 
   const [allItems, setAllItems] = useState([])
 
+  const [showCustomPrice, setShowCustomPrice] = useState({
+    standard: false,
```
Comment (olivermrbl) [inline src/domain/orders/details/returns/index.js:36]: Is this used?

File: src/domain/orders/details/returns/index.js [hunk 2]
```diff
@@ -30,12 +32,27 @@ const ReturnMenu = ({ order, onReturn, onDismiss, toaster }) => {
...
+    return: false,
+  })
+  const [customOptionPrice, setCustomOptionPrice] = useState({
+    standard: 0,
```
Comment (olivermrbl) [inline src/domain/orders/details/returns/index.js:40]: And this one?

File: src/domain/orders/details/swap/create.js [hunk 3]
```diff
@@ -87,6 +89,14 @@ const SwapMenu = ({ order, onCreate, onDismiss, toaster }) => {
   const [shippingPrice, setShippingPrice] = useState()
   const [noNotification, setNoNotification] = useState(order.no_notification)
   const [searchResults, setSearchResults] = useState([])
+  const [showCustomPrice, setShowCustomPrice] = useState({
+    standard: false,
```
Comment (olivermrbl) [inline src/domain/orders/details/swap/create.js:93]: Same as above

File: src/domain/orders/details/swap/create.js [hunk 4]
```diff
@@ -87,6 +89,14 @@ const SwapMenu = ({ order, onCreate, onDismiss, toaster }) => {
...
+    return: false,
+  })
+  const [customOptionPrice, setCustomOptionPrice] = useState({
+    standard: 0,
```
Comment (olivermrbl) [inline src/domain/orders/details/swap/create.js:97]: Same as above

**Review by olivermrbl — CHANGES_REQUESTED** [review]

File: src/domain/orders/details/returns/index.js (hunk 1 above)
Comment (sebastiannicolajsen) [reply src/domain/orders/details/returns/index.js:36]: Yes - used to either display the custom price for shipping or not (and determining to include it in the request)

File: src/domain/orders/details/returns/index.js (hunk 2 above)
Comment (sebastiannicolajsen) [reply src/domain/orders/details/returns/index.js:40]: Also used, for storing a custom price for shipping if needed

File: src/domain/orders/details/swap/create.js (hunk 3 above)
Comment (sebastiannicolajsen) [reply src/domain/orders/details/swap/create.js:93]: Same goes for this one

File: src/domain/orders/details/swap/create.js (hunk 4 above)
Comment (sebastiannicolajsen) [reply src/domain/orders/details/swap/create.js:97]: Same goes for this one

File: src/domain/orders/details/returns/index.js (hunk 1 above)
Comment (olivermrbl) [reply src/domain/orders/details/returns/index.js:36]: I am referring to the `standard` property for all comments to the PR. Looking at the files, it does not really seem to be used, but can you please confirm? Cause, then i will delete

Comment (olivermrbl) [reply src/domain/orders/details/returns/index.js:36]: @sebastiannicolajsen

**Comment by olivermrbl** [comment]
Solved in #325

**Pull request closed without merging**